src/components/Admin/QuestionDisplay.js

```diff
@@ -0,0 +1,212 @@
+import React, { Component, Fragment } from "react";
+import styled from "styled-components";
+
+import Button from "react-bootstrap/Button";
+import Col from "react-bootstrap/Col";
+import Row from "react-bootstrap/Row";
+
+import { withFirebase } from "upe-react-components";
+
+import Loader from "../Loader";
+import EditQuestion from "./EditQuestion";
+import { Container } from "../../styles/global";
+
+const StyledCol = styled(Col)`
+  padding: 10px 10px 10px 10px;
+`;
+
+const StyledHr = styled.hr`
+  border: 2px solid #333;
+  border-radius: 5px;
+`;
+
+const StyledDiv = styled.div`
+  width: 90%;
+  padding: 15px;
+  box-shadow: 0 3px 6px rgba(0, 0, 0, 0.16), 0 3px 6px rgba(0, 0, 0, 0.23);
+  border-radius: 15px;
+  text-align: center;
+  &:hover {
+    -webkit-transform: translateY(-5px);
+    transform: translateY(-5px);
+    transition: all 0.3s linear;
+  }
+
+  a {
+    color: white;
+    font-weight: bold;
+    padding-top: 10px;
+    padding-bottom: 10px;
+    border-bottom: 1px solid grey;
+  }
+
+  a[aria-current="page"] {
+    color: ${(props) => props.theme.palette.mainBrand};
+  }
+`;
+
+class QuestionDisplay extends Component {
```

Comment:
Converting to a function component will reduce a lot of lines of code and improve readability

Reply:
Resolved in commit 328335366bf582d828349cde311f9f15794bbca6